helm/cluster-cloud-director/templates/_helpers.tpl

```diff
@@ -124,6 +124,7 @@ joinConfiguration:
       node-labels: "giantswarm.io/node-pool={{ .pool.name }},{{- include "labelsByClass" . -}}"
     {{- include "taintsByClass" . | nindent  4}}
 files:
+{{- include "registryFiles" . | nindent 2 }}
```

Comment:
Can we add a condition `{{- if .Values.registry.configure -}}` ?

Reply:
The if the condition is in the function. https://github.com/giantswarm/cluster-cloud-director/pull/79/files#diff-7188efb4466b71c05057ba8992c443f8c9ab6bc2e1d67049a72f6a323aae2bdcR240

This line will be empty when registry.configure is not true.